ci/jenkins/test.sh

```diff
@@ -666,6 +834,40 @@ function run_conformance_windows {
     fi
 }
 
+function run_conformance_windows_containerd {
+    echo "====== Running Antrea Conformance Tests ======"
+    export GO111MODULE=on
+    export GOPATH=${WORKDIR}/go
+    export GOROOT=/usr/local/go
+    export GOCACHE=${WORKDIR}/.cache/go-build
+    export PATH=$GOROOT/bin:$PATH
+
+    if [[ "$TESTCASE" == "windows-networkpolicy-process" ]]; then
+        # Antrea Windows agents are deployed with scripts as processes on host for Windows NetworkPolicy test
+        wait_for_antrea_windows_processes_ready
+    else
+        # Antrea Windows agent Pods are deployed for Windows Conformance test
+        clean_for_windows_install_cni
+        wait_for_antrea_windows_pods_ready
+    fi
+
+    echo "====== Run test with conformance test ======"
+    export KUBE_TEST_REPO_LIST=${WORKDIR}/repo_list
+    if [ "$TESTCASE" == "windows-containerd-networkpolicy" ]; then
+        ginkgo -timeout=2h --no-color $E2ETEST_PATH -- --provider=skeleton --ginkgo.focus="$WINDOWS_NETWORKPOLICY_FOCUS" --ginkgo.skip="$WINDOWS_NETWORKPOLICY_CONTAINERD_SKIP" > windows_conformance_result_no_color.txt || true
+    else
+        ginkgo --no-color $E2ETEST_PATH -- --provider=skeleton --node-os-distro=windows --ginkgo.focus="$WINDOWS_CONFORMANCE_FOCUS" --ginkgo.skip="$WINDOWS_CONFORMANCE_SKIP" > windows_conformance_result_no_color.txt || true
```

Comment:
We can not specify `--node-os-distro=windows` in windows-containerd-networkpolicy or windows-networkpolicy, it is because most of the network policy test cases we expect to run in kubernetes test suite have the mark "[LinuxOnly]", and the parameter will lead the cases are passed. Although kubernetes mark these case to run on Linux only, they are supposed to work on Windows since Antrea Windows supports NP functions.

Reply:
better to add a comment for above explanation.

Reply:
Added comments.